tensorflow_lite_support/python/task/vision/core/utils.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Please hold on to that one: an equivalent will be checked in (as python/task/core/task_utils.py) very soon, you will be able to use that one instead.